woocommerce/woocommerce
# Fix product variation tooltip

### All Submissions:






### Changes proposed in this Pull Request:

This PR updates the copy of the default form values tooltip shown in the variable product meta box and also fixes the tooltip initialization when no variations exist yet.

Closes #33552 .

### How to test the changes in this Pull Request:

1. Load this PR and do a build, make sure the `woocommerce/client/legacy` is freshly build, I had to run `pnpm -- turbo run build --filter=woocommerce/client/legacy --force` to make sure the changes propagated instead of it using the cached version.
2. Go to **Products > Attributes** and add some attributes with terms
3. Go to **Products > Add New** and change the product to **Variable product**
4. Go to the **Attributes** tab and add the attribute you created in step 2, make sure to select **Used for variations** and click **Save attributes** after (you have to add values here)
5. Go to the **Variations** tab and click **Go** for the **Add variation** selection.
6. It should now show a **Default Form Values** with a tooltip after (as shown in the screenshot), the tooltip should work as expected
7. The tooltip should say: `Choose a default form value if you want a certain variation already selected when a user visits the product page`

<img width="1615" alt="Screen Shot 2022-07-05 at 4 31 11 PM" src="https://user-images.githubusercontent.com/2240960/177401848-ff7f9c3b-efc1-494e-a9aa-d247e726cab9.png">


### Other information:




### FOR PR REVIEWER ONLY:
